src/main/java/org/jenkinsci/test/acceptance/po/LdapSecurityRealm.java

```diff
@@ -62,7 +120,11 @@ protected WebElement find(String caption) {
      */
     public void configure(LdapDetails<T> ldapDetails) {
         ldapServer.set(ldapDetails.getHostWithPort());
-        advanced.click();
+        if(this.version116) {
```

Comment:
@fcojfernandez the convention here seems to be `if (this.version116) {` (space after `if`)